bindings/java/impl/src/main/com/apple/foundationdb/FDBTransaction.java

```diff
@@ -96,7 +97,7 @@ public void setReadVersion(long version) {
 		///////////////////
 		@Override
 		public AsyncIterable<KeyValue> getRange(KeySelector begin, KeySelector end, int limit, boolean reverse,
-		                                        StreamingMode mode) {
+												StreamingMode mode) {
```

Comment:
I think this change probably should be reverted (as the idea here is to use spaces here for alignment rather than tabs for indentation).